src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -133,7 +133,7 @@ void ContextAttribute::bsonAppendAttrValue
       break;
 
     default:
-      LM_E(("Runtime Error (unknown attribute type: %d)", valueType));
+      LM_E(("Runtime Error (unknown attribute type: %d on attribute %s)", valueType, name.c_str()));
```

Comment:
```suggestion
      LM_E(("Runtime Error (unknown attribute value type: %d on attribute %s)", valueType, name.c_str()));
```

To homogenize all these similar LM_E traces.